CMakeLists.txt

```diff
@@ -66,6 +66,10 @@ else()
     endif()
 endif()
 
+find_package(VulkanUtilityLibraries CONFIG REQUIRED QUIET)
+
+target_compile_definitions(Vulkan::LayerSettings INTERFACE VK_ENABLE_BETA_EXTENSIONS) # Enable beta Vulkan extensions
```

Comment:
Is this needed? Shouldn't line 39 handle that?

Reply:
Removed, it seems to work so good for me \o/